hugo/layouts/partials/header.html

```diff
@@ -12,10 +12,32 @@
             <div class="collapse navbar-collapse justify-content-end" id="navbarNavDropdown">
                 <ul class="navbar-nav">
                     <li class="nav-item">
-                        <a class="nav-link" href="engine">source{d} Engine</a>
+                        <a class="nav-link" href="why-sourced">Why source{d}</a>
                     </li>
                     <li class="nav-item">
-                        <a class="nav-link" href="enterprise">Enterprise</a>
+                        <a class="nav-link" href="features">Features</a>
+                    </li>
+                    <li class="nav-item dropdown">
+                        <a class="nav-link dropdown-toggle" href="enterprise" id="navbarDropdownMenuLinkEnt" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                            Enterprise
+                        </a>
+                        <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLinkEnt">
+                            <a class="dropdown-item" href="enterprise#solutions">Application Portfolio Management</a>
```

Comment:
All four links inside `Enterprise` menu points to the same section.
Is it correct?